cognite_toolkit/_cdf_tk/apps/_modules_app.py

```diff
@@ -52,7 +45,7 @@ def init(
             typer.Option(
                 "--clean",
                 "-a",
-                help="Clean target 'modules' directory if it exists",
+                help="Clean target 'organization' directory if it exists",
```

Comment:
The code deletes orgdir/modules, not orgdir

```suggestion
                help="Clean target directory if it exists",
```